hackathons/virtual_2.0.html

```diff
@@ -0,0 +1,1034 @@
+       </div>
+
+       <style>
+
+       .question h4 {
+         font-family: 'Oswald', sans-serif;
+       }
+       </style>
+
+
+       <div class="row">
+         <div class="col 12">
+
+         </div>
+       </div>
+
+
+    </div>
+  </div>
+  <div class="seperator-top hide-on-small-only" style="margin-top: -50px; margin-bottom: -1px;border-bottom: 51px solid #62a337; border-right: 1500px solid #303030; overflow: hidden; -webkit-transform: scaleX(1); transform: scaleX(1);"></div>
+
+  <div style="width: 100%; background-color: #62a337; height: 30px; margin-top: -10px;"></div>
+
+
+  <div class="section" style="width: 100%; background-color: #fff; padding: 40px 0px">
+    <div class="container">
+      <div class="row">
+         <div class="col s12">
+           <h2 style="color: #303030; margin-bottom: 18px; margin-top: 30px; text-align: center;"><i class="fas fa-user-plus" aria-hidden="true"></i>Interested in attending?</h2>
+           <h3 align="center">
+           	Subscribe for our Newsletter to know the further Announcements.
+           </h3>
+           
+           	<a href="https://defhacks.us17.list-manage.com/subscribe?u=6455f6de7b9e084b1f389c982&id=d2f6669d39" target="_blank">
+           	<input type="submit" name="subscribe" value="Subscribe" style="width:100px;margin:0 50%;position:relative;left:-50px; padding: 15px; background: green;">
+           	</a>
+           
+
+           
+           <!--
+
+           <p style="color: #303030; font-weight: 500; margin-bottom: 38px;text-align: center;">Registrations close on June 19th - so don't delay it! The registration only take a few minutes to complete. Be sure to read the <a href="http://mlh.io/code-of-conduct">Code of Conduct</a> as well! Remember, you won't have to register as a team, just register individually. When it's time to submit your project, you can add the names of all your teammates there!
+           </p><div class="press-block">
+              <div class="row center-align" style="margin: auto; width: 70%">
+                <div class="col s12" style="font-weight: 500!important; margin: auto;">
+                  <a class="press-btn waves-effect btn-flat modal-trigger" href="https://forms.gle/Y3vCpE2UeQyi85Qz9" target="_blank" style="border-color: #62a337; color: #fff; background-color: #62a337; margin-top: 18px; padding: 2px 25px; font-size: 1.5em;">HACKER REGISTRATION</a>
+                </div>
+                <div class="col s12" style="font-weight: 500!important; margin: auto;">
+                  <a class="press-btn waves-effect btn-flat modal-trigger" href="https://forms.gle/6JbW5BFbreJtN7e29" target="_blank" style="border-color: #62a337; color: #fff; background-color: #62a337; margin-top: 18px; padding: 2px 25px; font-size: 1.5em;">MENTOR/JUDGE SIGN-UP</a>
+                </div>
+           </div>
+         </div> -->
+       </div>
+     </div>
+   </div>
+  </div>
+
+  <div style="width: 100%; background-color: #62a337; height: 10px"></div>
```

Comment:
comment out hackathon schedule, guest speaker, submission tracks, event sponsors

Reply:
and comment out head organizers and judges